### PR TITLE
Add auto assign milestone GitHub action

### PR DESCRIPTION
**Pull request by jjw24**
- Added auto assign milestone GitHub action
- Fixed gitStream screenshot request configuration issue

### CODE REVIEW

File: .github/workflows/pr_milestone.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  automation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: set-milestone
+        uses: andrefcdias/add-to-milestone@v1.3.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          milestone: "+([0-9]).+([0-9]).+([0-9])"
```
[the remark this answers is not in the document]
Comment (jjw24) [reply .github/workflows/pr_milestone.yml:18]: Intentionally excluding 'Future' milestone because it will be a manual assignment.

File: .cm/gitstream.cm [hunk 2]
```diff
@@ -72,4 +72,8 @@ changes:
   # Sum all the line removed in the PR
   deletions: {{ branch.diff.files_metadata | map(attr='deletions') | sum }}
   # Calculate the ratio of new code
-  ratio: {{ (changes.additions / (changes.additions + changes.deletions)) * 100 | round(2) }}
\ No newline at end of file
+  ratio: {{ (changes.additions / (changes.additions + changes.deletions)) * 100 | round(2) }}
+
+has:
+  screenshot_link: {{ pr.description | includes(regex=r/!\[.*\]\(.*(jpg|svg|png|gif|psd).*\)/) }}
```
Comment (taooceros) [inline .cm/gitstream.cm:78]: what's the purpose of these changes?

**Review by taooceros — APPROVED** [review]

File: .cm/gitstream.cm (hunk 2 above)
Comment (jjw24) [reply .cm/gitstream.cm:78]: This is for gitStream, a custom function in the config I missed, resulting in gitStream bot commenting screenshot request despite already included one in PR description.
